vignettes/introduction-to-partition.Rmd

```diff
@@ -49,13 +49,15 @@ knitr::opts_chunk$set(
   warning = FALSE,
   message = FALSE
 )
+
+options(tibble.max_extra_cols = 10)
 ```
 
 ## Introduction to the partition package
 
-partition is a fast and flexible data reduction framework [@R-partition] that uses an approach called Direct-Measure-Reduce to create agglomerative partitions that maintain the user-specified minimum level of information. Each variable starts of as its own cluster; potential reduced variables are assessed by making sure this level of information is maintained. The reduced variables are also interpretable: original variables map to one and only one variable in the reduced data set. partition is flexible, as well: how variables are selected to reduce, how information loss is measured, and the way data is reduced can all be customized. 
+partition is a fast and flexible data reduction framework for R [@R-partition]. There are many approaches to data reduction, such as principal components analysis (PCA) and hierarchical clustering (both supported in base R). As opposed to these approaches, partition attempts to create a reduced data set that is both interpretable (each raw variable maps to one and only one reduced variable) and information-rich (reduced variables must meet an information threshold). Reducing the data this way often results in a data set that has a mix of raw features from the original data and reduced features.
 
-partition is particularly useful for highly correlated data, such as genetic data, where there is a lot of redundancy in the information each variable lends. `simulate_block_data()` simulates data like this: blocks of correlated data that are themselves independent of the other blocks in the data:
+partition is particularly useful for highly correlated data, such as genetic data, where there is a lot of redundancy in the information each variable lends. `simulate_block_data()` simulates data like this: blocks of correlated data that are themselves independent of the other blocks in the data.
```

Comment:
Suggestions:
partition is particularly useful for highly correlated data, such as genomic data, where there is a lot of redundancy. A simple model of, say, gene expression data could be block correlated Gaussian variables, which can be simulated with the function `simulate_block_data()`.